ranger-plugin/.gitignore

```diff
@@ -1,2 +1,3 @@
 target/
 Makefile.global
+/bin/
```

Comment:
is there any dir "bin" generated by building ? "/bin/" only ignore directory "bin" in current directory, e.g ranger-plugin/bin. Recommend "bin/" to ignore all "bin" directory from current directory to all sub-directories.